doc/source/development/maintaining.rst

```diff
@@ -449,9 +449,13 @@ which will be triggered when the tag is pushed.
     git tag -a v1.5.0.dev0 -m "DEV: Start 1.5.0"
     git push upstream main --follow-tags
 
-3. Build the source distribution (git must be in the tag commit)::
+3. Download the source distribution and wheels from the `wheel staging area <https://anaconda.org/scientific-python-nightly-wheels/pandas>`_.
+   Be careful to make sure that no wheels are missing (e.g. due to failed builds).
```

Comment:
Would be good to describe how we can tell if a build failed

Reply:
You kind of have to look manually for the individually jobs unfortunately. 

I'm trying to build reporting for failed jobs here
https://github.com/pandas-dev/pandas/issues/55903